hack/ci-e2e-test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# CI E2E Test Script
+# This script is called by the OpenShift CI system to run e2e tests on a remote server.
+#
+# Required environment variables:
+#   SHARED_DIR - Directory containing shared CI artifacts (server-ip file)
+#   CLUSTER_PROFILE_DIR - Directory containing cluster profile (packet-ssh-key)
```

Comment:
Maybe it is a good idea to make this script sensible for local testing as well. What do you think about taking the IP & SSH key instead ?